taxcalc/functions.py

```diff
@@ -989,15 +1024,17 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517,
     c63000 = c62800 - c62900
     c63100 = _taxbc - e07300 - c05700
     c63100 = c63100 + e10105
-    c63100 = max(0., c63100)
+    c63100 = max(0., c63100) + NIIT
```

Comment:
I believe we have discussed this change with Dan via email a while ago, and we proposed that `NIIT` should be added to `c63100`.